### ISSUE TITLE
Set the ''target aggregates'' view as the default page for the target tab for the supervisor (improvement for the issue #4840)

### ISSUE THREAD

**Issue by maremegaye** [medic/cht-core#6283]
When the target tab is opened, we have the labels repeated: targets and target aggregates as highligted in this below screenshot.
<img width="609" alt="Capture d’écran 2020-03-10 à 14 49 16" src="https://user-images.githubusercontent.com/30501949/76324898-9200a900-62de-11ea-8fc8-1cdfd5ee072a.png">

To avoid having the labels repeated, we would like to have the label "target aggregates" as the default page for the target tab for the supervisor. In that case, when the user needs to display the label "target" he will just select it from the dropdown 

**See the proposed views below**

**Default view: target aggreates**
<img width="604" alt="Capture d’écran 2020-03-10 à 14 55 04" src="https://user-images.githubusercontent.com/30501949/76325485-41d61680-62df-11ea-82f8-af7a97ec217c.png">

**View when the label "target" is selected**
<img width="590" alt="Capture d’écran 2020-03-10 à 14 58 03" src="https://user-images.githubusercontent.com/30501949/76325790-a6917100-62df-11ea-964b-8a531d5eedeb.png">

cc @michaelkohn

**Comment by dianabarsan** [comment]
@maremegaye 
I'm not sure I understand this correctly. 
We have 3 pages: 
1. The analytics landing page 
2. The targets page
3. The aggregates targets page

The analytics landing page is now displayed when clicking on the header link. 
The targets and aggregate targets page are linked to from the analytics landing page and the dropdown menu.

You are requesting that the header menu links to the aggregates page directly and that the dropdown menu link for "Targets" links to the Analytics landing page instead of the actual Targets page?

**Comment by maremegaye** [comment]
@dianabarsan what I am requesting is:

- when users click on the main target page (the same line with the people, message, report tabs) they see automatically the data for the aggregate target
- we will still have the "analytics landing page" with the same labels we have
- then when users select "target" from the "analytics landing page" they see the data related to the supervisor's target. What we need to remove from that view are the labels "my targets" and "CHW targets" and display the data automatically.

Please let me know if it is clear. I will update the description to make it more clear as well.

**Comment by maremegaye** [comment]
@dianabarsan he is the workflow recommended.
<img width="604" alt="Capture d’écran 2020-03-10 à 14 55 04" src="https://user-images.githubusercontent.com/30501949/77070126-96654a00-69e1-11ea-8130-37dcd93ddf8a.png">

**Comment by maremegaye** [comment]
@dianabarsan here is the workflow recommended
1- the supervisor opens the target tab and sees the default page which is the ''aggregate targets'' data
<img width="604" alt="Capture d’écran 2020-03-10 à 14 55 04" src="https://user-images.githubusercontent.com/30501949/77070291-df1d0300-69e1-11ea-923d-b0cfb5761a2e.png">

2- to view other data, the supervisor will click on the dropdown tab to select the page ''targets''
<img width="894" alt="Capture d’écran 2020-03-19 à 12 59 05" src="https://user-images.githubusercontent.com/30501949/77070463-2c00d980-69e2-11ea-9a6c-b38b75330a96.png">

Basically, what we want to delete are the two boxes for target aggregates and targets
<img width="317" alt="Capture d’écran 2020-03-19 à 13 07 10" src="https://user-images.githubusercontent.com/30501949/77070759-b6493d80-69e2-11ea-937c-27ed28510a59.png">

cc @n-orlowski
